app/models/ems_cluster.rb

```diff
@@ -288,31 +288,7 @@ def hosts_enabled_for_perf_capture
     hosts(:include => [:taggings, :tags]).select(&:perf_capture_enabled?)
   end
 
-  cache_with_timeout(:node_types) do
-    if !openstack_clusters_exists?
-      :non_openstack
-    elsif non_openstack_clusters_exists?
-      :mixed_clusters
-    else
-      :openstack
-    end
-  end
-
-  def self.openstack_clusters_exists?
-    ems = ManageIQ::Providers::Openstack::InfraManager.pluck(:id)
-    ems.empty? ? false : EmsCluster.where(:ems_id => ems).exists?
-  end
-
-  def self.non_openstack_clusters_exists?
-    ems = ManageIQ::Providers::Openstack::InfraManager.pluck(:id)
-    EmsCluster.where.not(:ems_id => ems).exists?
-  end
-
-  def openstack_cluster?
-    ext_management_system.class == ManageIQ::Providers::Openstack::InfraManager
-  end
-
   def self.display_name(number = 1)
-    n_('Cluster / Deployment Role', 'Clusters / Deployment Roles', number)
+    n_('Cluster', 'Clusters', number)
```

Comment:
This is why the vmware provider is failing, the display name of the cluster used to be "Cluster / Deployment Role" but now it is just "Cluster".

We are looking up the name of the cluster [here](https://github.com/ManageIQ/manageiq-providers-vmware/blob/ac3f7bf53d90eb2575b81fd2235b99aab8e75acd/app/models/manageiq/providers/vmware/infra_manager/inventory/parser.rb#L264)

Reply:
Love the cross-repo tests!

Reply:
Strictly speaking, the vmware provider is failing because of the equivalent change in `en.yml` below. I still haven't switched the behavior from `en.yml` to use `display_name` in particular models (still TODO).

Reply:
@mzazrivec we should probably include the 'Default for' in the translation right?

Reply:
@agrare I'm not quite sure I understand the question ...

Reply:
@mzazrivec nevermind I'll open a PR for vmware and we can discuss there